### ISSUE TITLE
tweaks for running in macOS

### ISSUE THREAD

**Issue by emilppp** [StottlerHenkeAssociates/Software-Architecture-Evaluation#2]
Hi! 
Following the installation instructions, using macOS Mojave, some tweaks had to be done to get the tool to run properly on my machine:

By default, using the Understand installer, Understand is installed in the Applications folder. Had to manually move the folder to `<current directory>/scitools` and then edit the `cloneAndAnalyze.sh` to instead point to
`scitools/Understand.app/Contents/MacOS/und` 
and
`scitools/Understand.app/Contents/MacOS/uperl`

Further, errors appeared when und tried to create a database. By manually creating the folder `<current directory>/created_dbs` this error was bypassed & now the tool runs perfectly fine :)

tldr:

- Copy/move Understand.app to `<current directory>/scitools`
- Change the `cloneAndAnalyze.sh` script to point to `und` and `uperl` that now resides in `<current directory>/scitools/Understand.app/Contents/MacOS/`
- Create a folder `<current directory>/created_dbs`

Thanks for a great tool! :)
